### PR TITLE
New UI, Search functionality & Overall Percentage 

### PR DESCRIPTION
**Pull request by Akshith-Patil**
**New UI design Implemented along with Search Bar & Overall Green Percentage**
1. Added new colored icons for status.
2. Changed font style for all text
3. Summary section with overall & repo count
4. Search functionality
5. Updated the internal/external/users view with new icons & fonts. 
6. Created separate CSS file for each component

Header/Summary
Summary   Header

Table View
Repo View

### CODE REVIEW

**Review by reteppeter — COMMENTED** [review]
Whilst each individual file looks fine, there needs to be an overall formatting pass (which could be done before merging) and refactoring (which shouldn't stop this from being merged; that's work for another sprint).

However, I don't agree with placing each pair of .tsx and .module.css files in their own folder - this just adds unnecessary clutter and takes up more room in a side bar, whilst not having any benefits for locality as two files with the same name but different extensions will be placed next to each other anyway.

File: components/SearchBar/SearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  searchTerm: any;
+  setSearchTerm: any;
+}) {
+  return (
+    <div className={styles.searchBarWidth}>
+      <div className={styles.searchBar}>
+        <input
+          className={styles.searchBar}
+          type="text"
+          placeholder="Search Repository..."
```
Comment (mt-empty) [inline components/SearchBar/SearchBar.tsx:16]: Just `Search...` should be enough

File: components/SummaryComponents/RepoOverviewTable/ReposOverviewTable.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import YellowIcon from "../../../components/images/yellowIcon.svg";
+import greenIcon from "../../../components/images/greenIcon.svg";
+import Image from "next/image";
+
+export default function ReposOverViewTable(props: { rankArray: any }) {
+  return (
+    <Table>
+      <TableRow>
+        <TableCell className={styles.totalsCellStyle}>
+          <h3>Total Repos</h3>
```
Comment (mt-empty) [inline components/SummaryComponents/RepoOverviewTable/ReposOverviewTable.tsx:15]: either `Total` or `Total repositories`

File: package-prev-commit.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (mt-empty) [inline package-prev-commit.json:1]: Please don't forget to remove this file before merging

**Review by mt-empty — CHANGES_REQUESTED** [review]
Good job guys,
I tested the GUI and the only thing that I found is:

Clicking on the question mark shows this:
image

My code review is not extensive, please ask someone to look at the code.

**Review by alexdelicato — APPROVED** [review]

**Review by mt-empty — APPROVED** [review]

**Pull request merged**